README.md

```diff
@@ -1,19 +1,24 @@
 # InviZible Pro
 
+Proudly presented by Garmatin Oleksandr, \
```

Comment:
@Gedsh How do you like this paragraph?

Reply:
Hello, this is a mistake, the application is not presented by its developer, but coded, why would you want to reduce the one who launched this project, carries it at arm's length and keeps it alive to a simple standard bearer? It is a nonsense in my opinion

Reply:
OK, I'll replace that with "coded", even though it doesn't flow as nicely in English.

Reply:
> OK, I'll replace that with "coded", even though it doesn't flow as nicely in English.

I agree, what do you think about :
"Proudly manufactured by"

Reply:
I think "made by" is better.